alfasim_sdk/hook_specs.py

```diff
@@ -146,6 +146,18 @@ def update_plugins_secondary_variables(ctx: "void*") -> "int":
     """
 
 
+def update_plugins_secondary_variables_extra(ctx: "void*") -> "int":
+    """
+    Internal simulator hook to update plugin's secondary variables in the ExtraVarSolver scope.
+    ExtraVarsolver is used to solve the tracer transport equation.
+    This is called as the last step on ALFAsim's ExtraVarSolver update variables workflow.
+
+    :param ctx:
+
+    :returns: Return OK if successful or anything different if failed
```

Comment:
This OK it's a macro?

Reply:
It is an `enum` from `alfasim-sdk\alfasim_sdk_api\common.h`